game/txt/Makefile

```diff
@@ -17,9 +17,15 @@ TXT=help.txt news.txt
 # below.)
 INDEX_FLAGS=
 
+CHANGES_FILES=$(wildcard ../../CHANGES*)
+HELP_CHANGES_FILES=$(patsubst ../../CHANGES.%,hlp/pennv%.hlp,$(CHANGES_FILES))
+
 all: $(TXT)
 
-help.txt: hlp/*.hlp hlp compose.sh
+$(HELP_CHANGES_FILES): hlp/pennv%.hlp: ../../CHANGES.%
+	../../utils/mkvershlp.pl hlp ../../CHANGES*
```

Comment:
It is, but it has to be installed. Also, @talvo and I talked and realized that there's already some voodoo in the Win build pipeline to build help.txt and friends, so calling this perl script as part of that should be no big deal (since we already have ActivePerl as a dependency on Windows).